<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NgocNV2-PC" name="Testcases.OpenIndividualCustomerTest" tests="49" failures="0" timestamp="15 Mar 2016 10:59:33 GMT" time="1070.990" errors="6">
  <testcase name="OpenIndividualCustomer" time="21.334" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.826" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="19.881" classname="Testcases.OpenIndividualCustomerTest">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:713)
	at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:734)
	at Testcases.SignInTest.switchChildId(SignInTest.java:97)
	at Testcases.SignInTest.navigateToVersion(SignInTest.java:86)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:171)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="22.475" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.337" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.451" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.448" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.698" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.734" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.889" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.422" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="20.112" classname="Testcases.OpenIndividualCustomerTest">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:713)
	at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:734)
	at Testcases.SignInTest.switchChildId(SignInTest.java:97)
	at Testcases.SignInTest.navigateToVersion(SignInTest.java:86)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:171)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="21.367" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.150" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.311" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.927" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.172" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.556" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.747" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.818" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="23.089" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.909" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.560" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.967" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="20.780" classname="Testcases.OpenIndividualCustomerTest">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:713)
	at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:734)
	at Testcases.SignInTest.switchChildId(SignInTest.java:97)
	at Testcases.SignInTest.navigateToVersion(SignInTest.java:86)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:171)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="21.871" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.497" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.243" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.455" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.773" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="31.205" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.572" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.291" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.257" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.549" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.579" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.589" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="20.780" classname="Testcases.OpenIndividualCustomerTest">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:713)
	at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:734)
	at Testcases.SignInTest.switchChildId(SignInTest.java:97)
	at Testcases.SignInTest.navigateToVersion(SignInTest.java:86)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:171)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="15.703" classname="Testcases.OpenIndividualCustomerTest">
    <error type="org.openqa.selenium.NoSuchElementException" message="Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//td/input[@id=&#039;transactionId&#039;][@type=&#039;hidden&#039;]&quot;}
Command duration or timeout: 10.01 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.52.0&#039;, revision: &#039;4c2593cfc3689a7fcd7be52549167e5ccc93ad28&#039;, time: &#039;2016-02-11 11:22:43&#039;
System info: host: &#039;NgocNV2-PC&#039;, ip: &#039;10.39.8.49&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_45&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=44.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 9d4723b3-8a65-420e-897d-9a4e210d052d
*** Element info: {Using=xpath, value=//td/input[@id=&#039;transactionId&#039;][@type=&#039;hidden&#039;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//td/input[@id='transactionId'][@type='hidden']"}
Command duration or timeout: 10.01 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'NgocNV2-PC', ip: '10.39.8.49', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=44.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 9d4723b3-8a65-420e-897d-9a4e210d052d
*** Element info: {Using=xpath, value=//td/input[@id='transactionId'][@type='hidden']}
	at sun.reflect.GeneratedConstructorAccessor28.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at Pages.OpenIndividualCustomerPage.txtCif(OpenIndividualCustomerPage.java:42)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:51)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//td/input[@id='transactionId'][@type='hidden']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'NgocNV2-PC', ip: '10.39.8.49', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_45'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/NgocNV2/AppData/Local/Temp/anonymous6583437536307531686webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10723)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/NgocNV2/AppData/Local/Temp/anonymous6583437536307531686webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:623)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="22.002" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="21.390" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.257" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="23.751" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.591" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.232" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="22.081" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="20.147" classname="Testcases.OpenIndividualCustomerTest">
    <error type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.LinkedHashMap$LinkedHashIterator.nextNode(LinkedHashMap.java:713)
	at java.util.LinkedHashMap$LinkedKeyIterator.next(LinkedHashMap.java:734)
	at Testcases.SignInTest.switchChildId(SignInTest.java:97)
	at Testcases.SignInTest.navigateToVersion(SignInTest.java:86)
	at Testcases.OpenIndividualCustomerTest.OpenIndividualCustomer(OpenIndividualCustomerTest.java:171)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- OpenIndividualCustomer -->
  <testcase name="OpenIndividualCustomer" time="21.958" classname="Testcases.OpenIndividualCustomerTest"/>
  <testcase name="OpenIndividualCustomer" time="24.257" classname="Testcases.OpenIndividualCustomerTest"/>
</testsuite> <!-- Testcases.OpenIndividualCustomerTest -->
